spark/v4.0/spark/src/test/java/org/apache/iceberg/spark/source/TestSparkTable.java

```diff
@@ -28,7 +29,9 @@
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.TestTemplate;
+import org.junit.jupiter.api.extension.ExtendWith;
 
+@ExtendWith(ParameterizedTestExtension.class)
```

Comment:
FYI that Spark 4 now made it into the codebase, so we should eventually also sync up 3.5 <-> 4.0 tests

Reply:
Okay got it. Thanks!